rdr_service/dao/bq_participant_summary_dao.py

```diff
@@ -284,6 +284,9 @@ def _prep_participant(self, p_id, ro_session):
         cohort_2_pilot_flag = \
             ParticipantCohortPilotFlag.COHORT_2_PILOT if cohort_2_pilot else ParticipantCohortPilotFlag.UNSET
 
+        # If RDR paired the pid to hpo TEST or flagged as either ghost or test participant, treat as test participant
+        test_participant = p.isGhostId == 1 or p.isTestParticipant == 1 or p.hpoId == TEST_HPO_ID
+
```

Comment:
Should this check the login_phone_number and email (if we have them)?

Reply:
If this doesn't identify the pid as a test participant, we will still end up calling `calculate_test_participant()` in the generator code later which can still flag it based on email or phone.   That hasn't been disabled, but there is a TODO to try and turn it off eventually.

Reply:
👍